tests/test_gates.py

```diff
@@ -63,9 +63,7 @@ def op(op_name):
         "ControlledQubitUnitary": qml.ControlledQubitUnitary(
             np.eye(2) * 1j, wires=[0], control_wires=[2]
         ),
-        "MultiControlledX": qml.MultiControlledX(
-            control_wires=[0, 1], wires=2, control_values="01"
-        ),
+        "MultiControlledX": qml.MultiControlledX(wires=(0, 1, 2), control_values="01"),
```

Comment:
I always worry about this assumed control vs target wire combination (makes life tricky for certain backend libs).
Not a comment for changing here though, just in general.

Reply:
Me too. FYI, this is the refactored PL code for initializing `wires` and `control_wires`, 
```
        if control_wires is None:
            if len(wires) > 1:
                control_wires = Wires(wires[:-1])
                wires = Wires(wires[-1])
            else:
                raise ValueError(
                    "MultiControlledX: wrong number of wires. 1 wire given. Need at least 2."
                )
        else:
            wires = Wires(wires)
            control_wires = Wires(control_wires)

            warnings.warn(
                "The control_wires keyword will be removed soon. Use wires = (control_wires,target_wire). See the documentation for more information.",
                category=UserWarning,
            )
```

Reply:
I think we plan on changing this! (cc @albi3ro)

Reply:
The call signature for `MultiControlledX` was changed recently to bring it in line with other gates like `CNOT`, `Toffoli`, `CRX`, etc.

The new signature will remain, but it will instead be a constructor for an operation of type `Controlled`.